src/main/resources/plugin.yml

```diff
@@ -7,6 +7,7 @@ author: RoboMWM
 authors: [Diemex, Big_Scary]
 #ensure that we are loaded after multiworld plugins
 softdepend: [Multiverse-Core, My_Worlds, MystCraft, Transporter, WorldGuard, Prism, LogBlock, CoreProtect, HawkEye, Crackshot]
+depend: [PlaceholderAPI]
```

Comment:
this is a yikes, cuz ppl like me don't use this plugin and thus will cause EHM to not load on such servers.